clang/include/clang/AST/Type.h

```diff
@@ -4689,6 +4704,7 @@ class FunctionType : public Type {
   struct alignas(void *) FunctionTypeArmAttributes {
     /// Any AArch64 SME ACLE type attributes that need to be propagated
     /// on declarations and function pointers.
+    LLVM_PREFERRED_TYPE(uint16_t)
```

Comment:
The preferred type attribute is for debugger behavior when the debugger would otherwise not show you helpful type information (e.g., the bit-field is `unsigned` but it holds an enumeration value). In this case, `AArch64SMETypeAttributes` might be more appropriate, but that enumeration is a bit mask and so maybe we don't need any preferred type at all? CC @Endilll for expertise on that.